src/templates/_stripView.hbs

```diff
@@ -1,7 +1,20 @@
+<script src="assets/scripts/client/jquery.min.js"></script>
+<script src="assets/scripts/client/jquery-ui.min.js"></script>
```

Comment:
I really question why you chose `jquery-ui`.  well, I can guess why, it used to be super common and its relatively quick to add drag and drop.

There are a few problems with that, though:
1) `jquery-ui` hasn't had a commit since September 2016, which suggests it is no longer maintained
1) all the browsers we support also support *native* drag and drop, no libraries needed (see: https://medium.com/quick-code/simple-javascript-drag-drop-d044d8c5bed5 or https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API)
1) we are slowly moving away from `jquery` entirely.  You just don't need it anymore and it adds additional weight to the code that we can live mostly without.